src/trace/layers/column_leaf/cursor.rs

```diff
@@ -117,6 +117,13 @@ where
         self.seek_key(key.0);
     }
 
+    #[inline]
+    fn seek_last(&mut self) {
```

Comment:
again no indication whether this succeeded?

Reply:
the method always succeeds, but if the iterator is empty, `key_valid()` will be false.  This is inline with the rest of the cursor API.